src/main/scala/mesosphere/marathon/MarathonModule.scala

```diff
@@ -191,6 +191,31 @@ class MarathonModule(conf: MarathonConf, http: HttpConf, zk: ZooKeeperClient)
       "MarathonScheduler")
   }
 
+  @Provides
+  @Singleton
+  def provideOfferReviveConf(): OfferReviverConf = conf
```

Comment:
is this better than `bind(classOf[OfferReviverConf]).toInstance(conf)` in MarathonModule#configure?